Forge.TreeWalker/contracts/ForgeTree.cs

```diff
@@ -192,6 +192,14 @@ public class RetryPolicy
         /// </summary>
         [DataMember]
         public long MaxBackoffMs { get; private set; }
+
+        /// <summary>
+        /// This setting is specifically for RetryPolicyType.FixedCount.
+        /// Max retry is the maximum number of times to attempt to run an action before failing. 
+        /// The default value is 0; if value is not specified no retries will run. 
+        /// </summary>
+        [DataMember]
+        public int MaxRetry { get; private set; } = 0;
```

Comment:
>MaxRetry [](start = 19, length = 8)

Consider naming this "MaxRetryCount" to more clearly convey that this is a fixed count of retries.